src/client/styles/modules/topnav-layout.less

```diff
@@ -63,3 +63,44 @@
     }
   }
 }
+
+div[data-dir='rtl'] .topnav-layout {
+  .left {
+    right: 0;
+    padding-right: 16px;
+    padding-left: 0px;
+  }
+
+  .right {
+    right: unset;
+    left: 0;
+    margin-right: 0px;
+    margin-left: 16px;
+  }
+
+  .center {
+    @media @small {
+      width: calc(100% ~'-' (@left-width + @right-width + 2 * @margin));
```

Comment:
looks like these calculations are the same as above within the normal `.topnav-layout .center`. Do you need them to be duplicated?

Reply:
Removed them.